tools/rebalance-corenrn-data.py

```diff
@@ -0,0 +1,207 @@
+#!/bin/env python3
+
+"""
+A post-processing script to redistribute CoreNeuron input files
+more evenly across ranks based on their filesystem size.
+
+Blue Brain Project - EPFL, 2024
+
+"""
+
+import argparse
+import heapq
+import itertools
+import logging
+import math
+import os
+import sys
+
+# Numpy may be required (histogram)
+numpy = None
+
```

Comment:
Maybe it makes sense to have some constants (for example):
```
PROGRESS_UPDATE_FREQUENCY = 50
DEFAULT_OUTPUT_FILE = "rebalanced-files.dat"
FILL_VALUE = "-1\n"
```
Or the names you think makes more sense